src/decaylanguage/dec/dec.py

```diff
@@ -154,7 +154,7 @@ def from_string(cls, filecontent):
         stream.seek(0)
 
         _cls = cls()
-        _cls._dec_file_names = "<dec file input as a string>"
+        _cls._dec_file_names = ["<dec file input as a string>"]
```

Comment:
Types should be consistent if possible; this is a list normally.

Reply:
Indeed I made a mistake. Should be a list.